src/ServiceControl.AcceptanceTests/AcceptanceTest.cs

```diff
@@ -89,6 +104,33 @@ public void Teardown()
             Trace.Listeners.Remove(textWriterTraceListener);
         }
 
+        void ResetSqlDb(string connectionString)
+        {
+            using (var connection = new SqlConnection(connectionString))
+            {
+                connection.Open();
+
+                var command = new SqlCommand(@"
+DECLARE @name NVARCHAR(128)
+DECLARE @schema NVARCHAR(128)
+DECLARE @SQL NVARCHAR(254)
+
+SELECT TOP 1 @name = sys.objects.name, @schema = sys.schemas.name FROM sys.objects INNER JOIN sys.schemas ON sys.objects.schema_id = sys.schemas.schema_id WHERE [type] = 'U' ORDER BY sys.objects.name
+
+WHILE @name IS NOT NULL
+BEGIN
+    SELECT @SQL = 'DROP TABLE ' + QUOTENAME(@schema) + '.' + QUOTENAME(RTRIM(@name))
+    EXEC (@SQL)
+    PRINT 'Dropped Table: ' + @schema + '.' + @name
+	SELECT  @name = NULL
+    SELECT TOP 1 @name = sys.objects.name, @schema = sys.schemas.name FROM sys.objects INNER JOIN sys.schemas ON sys.objects.schema_id = sys.schemas.schema_id WHERE [type] = 'U' ORDER BY sys.objects.name
+END
+", connection);
+
+                command.ExecuteNonQuery();
```

Comment:
To delete all the tables it should be an easy 2 liner:

```csharp
            using (var connection = new SqlConnection(sqlDbConnectionString))
            {
                var dropConstraints = "EXEC sp_msforeachtable 'ALTER TABLE ? NOCHECK CONSTRAINT all'";
                var dropTables = "EXEC sp_msforeachtable 'DROP TABLE ?'";

                await connection.OpenAsync().ConfigureAwait(false);
                await connection.ExecuteAsync(dropConstraints).ConfigureAwait(false);
                await connection.ExecuteAsync(dropTables).ConfigureAwait(false);
            }
```

Reply:
changed